dask/_version.py

```diff
@@ -16,7 +16,7 @@
 import re
 import subprocess
 import sys
-from typing import Callable, Dict
+from typing import Callable
```

Comment:
```suggestion
from collections.abc import Callable
```

Reply:
pyupgrade doesn't upgrade `typing.Callable` to  `collections.abc.Callable` because it is broken in python 3.9.0 and 3.9.1 https://github.com/asottile/pyupgrade/issues/677

typing.Callable is deprecated in Python 3.9 and I suspect nobody is using these version so I upgraded these manually, we should also consider adding `from __future__ import annotations` everywhere with isort so the broken codepath on 3.9.0 and 3.9.1 isn't run

Reply:
> pyupgrade doesn't upgrade typing.Callable to collections.abc.Callable because it is broken in python 3.9.0 and 3.9.1 https://github.com/asottile/pyupgrade/issues/677

I'm unconcerned about this; I'm strongly in favour of supporting only the latest patch version

> we should also consider adding from __future__ import annotations everywhere with isort

Indeed. This is already happening in distributed.

Reply:
I will make the isort change in a followup PR